lib/install.ps1

```diff
@@ -679,14 +679,16 @@ function create_shims($manifest, $dir, $global, $arch) {
         $target, $name, $arg = shim_def $_
         write-output "Creating shim for '$name'."
 
-        # check valid bin
-        $bin = "$dir\$target"
-        if(!(is_in_dir $dir $bin)) {
-            abort "Error in manifest: bin '$target' is outside the app directory."
+        if(test-path "$dir\$target") {
+            $bin = "$dir\$target"
+        } elseif(test-path $target) {
```

Comment:
Not sure what this check is for (if $target is absolute?)

Reply:
Yeah, just because absolute path is better than invalid path.